### PR TITLE
feat: [MSSQL] enable changing storage type

### PR DESCRIPTION
**Pull request by fnaranjo-vmw**
[#185149472](https://www.pivotaltracker.com/story/show/185149472)

Implementation in https://github.com/cloudfoundry/csb-brokerpak-aws/pull/1060

Which other branches should this be merged with (if any)? none

### CODE REVIEW

File: reference/aws-mssql.html.md.erb [hunk 1]
```diff
@@ -148,6 +148,28 @@ provision only:
       <td>None</td>
       <td>provision and update</td>
     </tr>
+    <tr>
+      <td><code>storage_type</code></td>
+      <td>String</td>
+      <td>
+        Type of storage to be used.
```
Comment (zucchinidev) [inline reference/aws-mssql.html.md.erb:155]:
```suggestion
        Type of storage to be used.
        One of <code>standard</code> (magnetic), <code>gp2</code> (general purpose SSD), <code>gp3</code>
        (general purpose SSD), or <code>io1</code> (provisioned IOPS SSD).
```

Comment (zucchinidev) [reply reference/aws-mssql.html.md.erb:155]: I’d rather add the code the client should use to make their life easier.

File: reference/aws-mssql.html.md.erb [hunk 2]
```diff
@@ -148,6 +148,28 @@ provision only:
...
+      <td>provision and update</td>
+    </tr>
+    <tr>
+      <td><code>iops</code></td>
+      <td>Integer</td>
+      <td>
+        The amount of provisioned IOPS.
+        For this property to take effect, <code>storage_type</code> must be set to <code>gp3</code> or <code>io1</code>.
+        For more information on customizing iops with gp3 check <a href="https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage.GeneralSSD">General Purpose SSD storage</a>.
+        For more information on customizing iops with io1 check <a href="https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS">Provisioned IOPS storage</a>.
```
Comment (zucchinidev) [inline reference/aws-mssql.html.md.erb:168]: praise: thanks for adding the two links.

**Review by zucchinidev — APPROVED** [review]

File: reference/aws-mssql.html.md.erb (hunk 1 above)
Comment (fnaranjo-vmw) [reply reference/aws-mssql.html.md.erb:155]: I don't understand what you mean.

**Pull request merged**